Lib9c/Action/BuyMultiple.cs

```diff
@@ -0,0 +1,353 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Nekoyume.Model.EnumType;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Mail;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Serilog;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionType("buy_multiple")]
+    public class BuyMultiple : GameAction
+    {
+        public Address buyerAvatarAddress;
+        public Address sellerAgentAddress;
+        public Address sellerAvatarAddress;
+        public IEnumerable<Guid> productIds;
+        public BuyerResult buyerResult;
+        public SellerResult sellerResult;
```

Comment:
Each product(shop item) has its own `sellerAgentAdress` and `sellerAvatarAddress`, `sellerResult`. How about writing it like below?
```
class BuyInfo
{
  Address agentAddress;
  Address avatarAddress;
  Guid productId;
}

List<BuyInfo> buyInfos;
List<SellerResult> sellerResults;

Execute()
{
  // buy process for `buyInfos`
  foreach (BuyInfo buyInfo in buyInfos)
  {
    // buy

    // add seller result to `sellerResults`
  }
}
```